README.md

```diff
@@ -1,185 +1,74 @@
-|../Interrupt	|CPU  	|Interrupt   Service Routine example using Eclipse* MRAA   	|
-|../Onboard-Blink    	|CPU  	|Built-in   LED blink for common IoT boards using Eclipse* MRAA 	|
-|../PWM 	|CPU  	|Pulse   Width Modulation pin output using Eclipse* MRAA   	|
-|../Up2   LEDs    	|CPU  	|Built-in   LED example for UP* Squared using Eclipse* MRAA	|
-|__Tools/Migration/__|
-|../folder-options-dpct |CPU| How to migrate more complex projects and use options for DPC++ Compatibility Tool (dpct) |
-|../rodinia-nw-dpct |CPU|How to migrate a Make/CMake project from CUDA to Data Parallel C++ using the Intel DPC++ Compatibility Tool. |
-|../vector-add-dpct |CPU|how to migrate a simple program from CUDA to Data Parallel C++ using the Intel DPC++ Compatibility Tool. |
-|__Tools/VTuneProfiler__/
-|../matrix_multiply_vtune|CPU|A sample containing multiple implementations of matrix multiplication. This sample code is implemented using DPC++ language for CPU and GPU.|
+- If you are using Visual Studio 2019, Visual Studio 2019 version 16.4.0 or newer is required.
+- Windows support for the FPGA code samples is limited to the FPGA emulator and optimization reports. Compile targets for FPGA hardware are provided on Linux only. See any FPGA code sample for more details.
+- If you encounter a compilation error when building a sample program, such as the example error below, the directory path of the sample may be too long. The workaround is to move the sample to a temp directory.
+- Example error: *Error MSB6003 The specified task executable dpcpp.exe could not be run .......
 
-## License
+## Known Issues or Limitations
 
-Code samples are licensed under the MIT license. See
-[License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
+## Contributing
 
-Third party program Licenses can be found here: [third-party-programs.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/third-party-programs.txt)
+See [CONTRIBUTING wiki](https://github.com/oneapi-src/oneAPI-samples/blob/master/CONTRIBUTING.md) for more information.
 
-## Known Issues or Limitations
 
-### On Windows Platform
-- If you are using Visual Studio 2019, Visual Studio 2019 version 16.4.0 or newer is required.
-- Windows support for the FPGA code samples is limited to the FPGA emulator and optimization reports. Compile targets for FPGA hardware are provided on Linux only. See any FPGA code sample for more details.
-- If you encounter a compilation error when building a sample program, such as the example error below, the directory path of the sample may be too long. The work around is to move the sample to a directory such as "c:\temp\sample_name".
-    - Example error: *Error MSB6003 The specified task executable "dpcpp.exe" could not be run .......*
 
-## Contribute
+## New Code Samples
+
+|Code Sample    |Supported Intel&reg;   Architecture(s)|Description|

```

Comment:
These headings do not match the data in the table.

Reply:
fixed in drop i just did